src/database/DataTypes/Item.js

```diff
@@ -224,6 +216,96 @@ export class Item extends Realm.Object {
   toString() {
     return `${this.code} - ${this.name}`;
   }
+
+  /**
+   * Finds all this items batches in a specific location.
+   * If no location is passed, all of this items ItemBatches
+   * are returned.
+   * @param {Location} location
+   */
+  getBatchesInLocation({ id } = {}) {
```

Comment:
id: locationId ? (i know the jdocs say it all, but seems more appropriate (not required for PR approval though

Reply:
done 7629c92aa4f52605295b355a311f7885d3ab49a1

Still trying to find my balance between explicit and implicit code, how clear etc.

Reply:
@joshxg the only reason i mentioned { locationId } is because of ambiguity of 'id' (all record have it)